Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -152,7 +152,7 @@
       <EnvironmentVariables>
          <EnvironmentVariable
             key = "OS_ACTIVITY_MODE"
-            value = "${DEBUG_ACTIVITY_MODE}"
+            value = ""
```

Comment:
Leaving `OS_ACTIVITY_MODE` without any value set allows us to see the logs in console.app on iOS devices